homeassistant/components/media_extractor/manifest.json

```
@@ -3,7 +3,7 @@
   "name": "Media extractor",
   "documentation": "https://www.home-assistant.io/components/media_extractor",
   "requirements": [
-    "youtube_dl==2019.06.27"
+    "youtube_dl==2019.07.01"
```

```suggestion
    "youtube_dl==2019.07.02"
```

There is no version 2019.07.01. Should be [2019.07.02](https://github.com/ytdl-org/youtube-dl/releases/tag/2019.07.02) according to [version.py](https://github.com/ytdl-org/youtube-dl/blob/2019.07.02/youtube_dl/version.py#L3).